contract/rust/src/lib.rs

```diff
@@ -35,13 +53,15 @@ impl Welcome {
     }
 }
 
+// the rest of this file involves testing
```

Comment:
for orientation comments like this, i would use a larger comment like

```
//----------------------------------------------------------------
// The rest of this file holds the inline tests for the code above
// 
// Rust allows you to include tests with your code
// see more here: https://doc.rust-lang.org/book/ch11-01-writing-tests.html
//----------------------------------------------------------------
```

Reply:
great idea, added this